releasenotes/notes/removed-ibmq-provider-1c757ce5ef01fb19.yaml

```
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    ``qiskit-ibmq-provider`` is deprecated and has been dropped as a requirement. Interactions
+    with backends should use the ``qiskit-ibm-provider`` package instead (see the 
```

Just clarifying that other providers are possible:

```suggestion
    with IBM backends should use the ``qiskit-ibm-provider`` package instead (see the 
```

Is it worth mentioning that qiskit-ibm-provider is not a new requirement and must be installed separately if needed?